public/index.html

```diff
@@ -3,24 +3,28 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
+<link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
Thanks @Charlotteis I improved the indentation on the index.html on the latest commit